lib/facility_maker.rb

```diff
@@ -0,0 +1,50 @@
+class FacilityMaker
+  attr_reader :add_facility, :facility, :add_facility, :facility_info, :name
+  attr_accessor :name, :address, :phone, :facilities, :add_facility
```

Comment:
This class has no attributes so we shouldn't need or have any attr readers or accessors